lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -104,14 +104,14 @@ typedef dynamic Dispatcher(dynamic action);
 ///     Store store1 = new Store<CounterState>(counterStateReducer, initialState: new CounterState(count: 0));
 ///     Store store2 = new Store<BigCounterState>(bigCounterStateReducer, initialState: new BigCounterState(bigCount: 100));
 ///
-///     UiFactory<CounterProps> ConnectedCounter = connect<CounterState, CounterProps>(
+///     UiFactory<CounterProps> Counter = connect<CounterState, CounterProps>(
```

Comment:
#supernit while we're here we could clean up the formatting a bit (same as the last example). 

Github needs to change the suggestion scope to be just a liiiittle wider. It's just a little too narrow here 😞